includes/emails/class-edd-email-tags.php

```diff
@@ -833,11 +833,9 @@ function edd_email_tag_payment_method( $payment_id ) {
  * Email template tag: sitename
  * Your site name
  *
- * @param int $payment_id
- *
  * @return string sitename
  */
-function edd_email_tag_sitename( $payment_id ) {
```

Comment:
If we're removing a parameter, we need to properly deprecate it.

Reply:
This parameter should never have been here. It never did anything, and was never used.